docs/src/theme/ReactLiveScope/index.js

```diff
@@ -1,5 +1,4 @@
 import ExecutionEnvironment from "@docusaurus/ExecutionEnvironment";
-// eslint-disable-next-line no-restricted-imports
```

Comment:
Why isn't this needed anymore? Does the rule to disable React barrel imports still work?

Reply:
This is not needed because `no-restricted-imports`set to `off` in documentation.

https://github.com/vivid-planet/comet/pull/3230/files#diff-1c7e12ff07ecefe2dd53afb58f8ed79d66dd840d6e97ddeb0641ff23d4639c11R12

Reply:
i will activate no-restricted-imports in this package. then it will work

Reply:
I believe this has already been done in #3231 🤔

Reply:
fixed.